### DISCUSSION TITLE
OCPP 1.6 Json

### DISCUSSION THREAD

**Discussion by taschaue** [citrineos/citrineos#40] [Q&A]
Hi,

I started testing citrineos, which is a very cool project for EV charging.
Does it support OCPP 1.6 Json? I tested it with the protocol parameter in the config and changed it to "ocpp1.6" but this did not work correctly. Do you have any advice?

Thanks
taschaue

**Comment by ChrisWeissmann** [comment]
Hi @taschaue, you are correct that 1.6 is currently not supported yet. We have been discussing that it will be part of the release cycle 1.7 of citrine. We know, missed opportunity for matching 1.6 of OCPP and 1.6 of citrine 😅. But we have some other things on our priority list before we got to it.

**Comment by yuvenus** [comment]
As of [CitrineOS release 1.6.0](https://github.com/citrineos/citrineos-core/releases/tag/v1.6.0) (we managed to make it match!), OCPP 1.6 is officially supported. 🎉
